src/Promitor.Scraper.Tests.Unit/Serialization/v1/Core/AggregationDeserializerTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.ComponentModel;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Core;
+using Xunit;
+
+namespace Promitor.Scraper.Tests.Unit.Serialization.v1.Core
+{
+    [Category("Unit")]
+    public class AggregationDeserializerTests
+    {
+        private readonly AggregationDeserializer _deserializer;
+
+        public AggregationDeserializerTests()
+        {
+            var logger = new Mock<ILogger>();
+
+            _deserializer = new AggregationDeserializer(logger.Object);
+        }
+
+        [Fact]
+        public void Deserialize_IntervalSupplied_SetsInterval()
+        {
+            // Arrange
+            const string yamlText =
+@"aggregation:
+    interval: 00:07:00";
+            DeserializerTestHelpers.AssertPropertySet(
```

Comment:
I prefer to create our own assert class so that it feels similar, we might already have one but feel free to move it to `YamlAsserts.AssertPropertySet` for examle

Reply:
Will do.